src/main/java/gregtech/api/capability/impl/FuelRecipeLogic.java

```diff
@@ -207,21 +213,90 @@ public long getMaxVoltage() {
         return maxVoltage;
     }
 
+    protected double calculateFuelConsumptionMultiplier() {
+        return 1.0;
+    }
+
     protected int calculateFuelAmount(FuelRecipe currentRecipe) {
-        return currentRecipe.getRecipeFluid().amount * getVoltageMultiplier(getMaxVoltage(), currentRecipe.getMinVoltage());
+        return (int) (currentRecipe.getRecipeFluid().amount * getVoltageMultiplier(getMaxVoltage(), currentRecipe.getMinVoltage()) * calculateFuelConsumptionMultiplier());
+    }
+
+    public int calculateFuelAmount() {
+        return calculateFuelAmount(this.previousRecipe);
+    }
+
+    protected double calculateRecipeDurationMultiplier() {
```

Comment:
This is policy that somebody can override and it will change the behaviour of the machine.
So calculateX() is correct.